### ISSUE TITLE
inject.js

### ISSUE THREAD

**Issue by zaahiral** [IdkDwij/Site#2]
nice file

**Comment by IdkDwij** [comment]
it does smt i dont remember

**Comment by zaaahir** [comment]
yeah try and steal data from anyone who clicks on the website link. disgusting.

**Comment by IdkDwij** [comment]
> yeah try and steal data from anyone who clicks on the website link. disgusting.

stop being autistic?
It does something thats all i remember but its not stealing shit
